AutomatedTesting/Gem/PythonTests/scripting/FileMenu_Default_NewAndOpen.py

```diff
@@ -4,25 +4,26 @@
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
 """
-from PySide2 import QtWidgets
 
-import editor_python_test_tools.pyside_utils as pyside_utils
+from PySide2 import QtWidgets
+from editor_python_test_tools.utils import TestHelper as helper
 from editor_python_test_tools.utils import Report
-
 import azlmbr.legacy.general as general
+import editor_python_test_tools.pyside_utils as pyside_utils
 
 
 # fmt: off
```

Comment:
this isn't needed the Tests class no longer uses misaligned formatting. you can remove the # fmt: off and the corresponding # fmt: on
those are pycharm marks to avoid syntax PEP8 compliance notifications

Reply:
removed the fmt on/off lines